### ISSUE TITLE
Server closes by itself after less than an hour.

### ISSUE THREAD

**Issue by yorkie98** [UniversalMediaServer/UniversalMediaServer#5205]
Hi,
UMS stops running and disappears from the sys tray after 30-45 minutes of operation.
Log attached.

Thanks,
Yorkie98.
[debug.log.prev.zip](https://github.com/user-attachments/files/18266764/debug.log.prev.zip)

**Comment by SubJunk** [comment]
@yorkie98 were there any other files in the folder? I don't see anything in here

**Comment by yorkie98** [comment]
![UMS files](https://github.com/user-attachments/assets/48191355-7e47-439f-874a-cc7af71376ba)
@SubJunk  These files were also in the folder.

**Comment by SubJunk** [comment]
Is this Warren Wilding on Facebook? I am just making sure I connect the info

If it is, you also said the server hasn't kept running for you since v12, so you have to keep reverting to v11 for stability?

I can see you are using a TV that we improved support for in v14.9.0 but I don't think that would improve that crashing problem.

I don't see anything to indicate a crash in those logs though. What files are in `C:\Program Files (x86)\Universal Media Server\`? Are there any that say "error" or "exception"? Sometimes if Java crashes, especially if it runs out of memory, it could leave a file in there

**Comment by MEGAKNIGHT88** [comment]
Have same issues here! Sending the latest debug file, as far as could possible on the fly for now..... Also have strange issue, when playing file sometimes, for ex. if the playback has started and then choose to stay on pause immediately, when resuming it will skips or switches to the next file immediately. It's already started some releases later time after time.
[debug.log](https://github.com/user-attachments/files/18285257/debug.log)

**Comment by yorkie98** [comment]
> Is this Warren Wilding on Facebook? I am just making sure I connect the info
> 
> If it is, you also said the server hasn't kept running for you since v12, so you have to keep reverting to v11 for stability?
> 
> I can see you are using a TV that we improved support for in v14.9.0 but I don't think that would improve that crashing problem.
> 
> I don't see anything to indicate a crash in those logs though. What files are in `C:\Program Files (x86)\Universal Media Server\`? Are there any that say "error" or "exception"? Sometimes if Java crashes, especially if it runs out of memory, it could leave a file in there

Yes, I'm the same person.
Unfortunately, I have totally uninstalled the newer version and I have rolled back to the last version which works reliably for me, 11.6.0, although this is not without its quirks.
The windows install I'm running on, is a few years old and has some history so I might try a clean install and see where that takes me.

**Comment by SubJunk** [comment]
@MEGAKNIGHT88 thanks for adding information here. If you want to make new issues for the other problems that would be good. If we start talking about multiple problems in one issue it gets hard to track and resolve.

With the logs you provided, did UMS close itself before you made them?

**Comment by SubJunk** [comment]
@yorkie98 I wonder if it is related to available RAM. I can see in the logs that the machine has 8GB of RAM, and when UMS was launched there was about 2GB free. That should be enough for UMS, but if the machine uses more RAM there might not be enough room.

As an example, right now this GitHub web page is using 112 MB of RAM, and that's a small, simple page. I have seen single tabs use 1GB, so they can easily add up and limit the memory for other programs.

**Comment by yorkie98** [comment]
> @MEGAKNIGHT88 thanks for adding information here. If you want to make new issues for the other problems that would be good. If we start talking about multiple problems in one issue it gets hard to track and resolve.
> 
> With the logs you provided, did UMS close itself before you made them?

Yes, that's correct, so the log I provided was after it had closed itself. Bearing in mind, this was when it was idle, it wasn't, not had it been in use.

**Comment by MEGAKNIGHT88** [comment]
> @MEGAKNIGHT88 thanks for adding information here. If you want to make new issues for the other problems that would be good. If we start talking about multiple problems in one issue it gets hard to track and resolve.
> 
> With the logs you provided, did UMS close itself before you made them?

I think yes it is, apparently so, because I send them, after the program was closed.

**Comment by SubJunk** [comment]
@MEGAKNIGHT88 I can see in both of those logs that something is triggering the a shut down:

```
INFO  2025-01-01 04:44:50.173 [UMS Shutdown] Shutting down UPnP service...
INFO  2025-01-01 04:44:50.728 [UMS Shutdown] UPnP service shutdown completed
```

Can you please attach your UMS.conf file too? Something is weird about those logs

**Comment by MEGAKNIGHT88** [comment]
Sure thing. 
[UMS config.zip](https://github.com/user-attachments/files/18295815/UMS.config.zip)

**Comment by SubJunk** [comment]
There is a setting in there that might be causing the log file to not catch the problem, `logging_buffered =true` should be `false`, also potentially `logging_syslog_facility =NEWS` could interfere.
Maybe you can rename your UMS.conf file while UMS is closed, which will cause UMS to use its defaults, and see if this problem still happens? You can still set shared content

**Comment by MEGAKNIGHT88** [comment]
Will check.

**Comment by MEGAKNIGHT88** [comment]
Reinstalled the program and reset the configuration file and playback is now working normally at the first start, at least, in this case. Seems to be something were mismatched in the settings or it has old entries. Much thanks and really appreciated! If'll find some other bugs, will report back. Happy New Year & Happy Holidays!

**Comment by SubJunk** [comment]
@yorkie98 I wonder if yours is similar, can we see your UMS.conf file?

**Comment by yorkie98** [comment]
> @yorkie98 I wonder if it is related to available RAM. I can see in the logs that the machine has 8GB of RAM, and when UMS was launched there was about 2GB free. That should be enough for UMS, but if the machine uses more RAM there might not be enough room.
> 
> As an example, right now this GitHub web page is using 112 MB of RAM, and that's a small, simple page. I have seen single tabs use 1GB, so they can easily add up and limit the memory for other programs.



> @yorkie98 I wonder if it is related to available RAM. I can see in the logs that the machine has 8GB of RAM, and when UMS was launched there was about 2GB free. That should be enough for UMS, but if the machine uses more RAM there might not be enough room.
> 
> As an example, right now this GitHub web page is using 112 MB of RAM, and that's a small, simple page. I have seen single tabs use 1GB, so they can easily add up and limit the memory for other programs.

I doubt there would have been much more RAM usage, the machine just runs a couple of apps and doesn't have a browser with loads of tabs running, it's pretty much left to do its thing, downloading/storing media, and serving it.
I could possibly look into increasing the RAM but I doubt that's the issue.

**Comment by yorkie98** [comment]
> @yorkie98 I wonder if yours is similar, can we see your UMS.conf file?

I'm now running an old version so I doubt the file will be of any help to you at this stage.
If you'd like, I can uninstall old version, re-install new and get the file when it crashes.

**Comment by SubJunk** [comment]
Unless you are changing settings when you go back to the old version it would still be useful, the conf file just stores your settings

**Comment by MEGAKNIGHT88** [comment]
On PS3, while resuming after sometime, it's quickly showing some kind of error, then switching on to the next file.

**Comment by Mik-S-UMS** [comment]
Could you post fresh logs with this and I'll have a look to see if anything is obvious.
Make sure you include all the logs following [these](https://www.universalmediaserver.com/forum/viewtopic.php?f=9&t=556) instructions.
